pkg/server/server_test.go

```diff
@@ -124,15 +130,15 @@ func TestMain(m *testing.M) {
 
 	grpcServer := server(dbStore)
 
-	fbcJsonStore, err := fbcJsonCache(fbcDir, filepath.Join(tmpDir, "json-cache"))
+	fbcJsonStore, err := fbcCache(fbcDir, filepath.Join(tmpDir, "cache"))
```

Comment:
```suggestion
	fbcStore, err := fbcCache(fbcDir, filepath.Join(tmpDir, "cache"))
```
?

Same for `fbcJsonDeprecationStore` --> `fbcDeprecationStore` ?

Reply:
Basically nits, but it seems like most variable names have had the `json` removed from them since now they can have protobuf backing instead. Given the variability, it would be nice to only have json in vars which are definitively of that type.

Reply:
Good catch! Done.